docs/flow/config-language/expressions/operators.md

```diff
@@ -6,3 +6,97 @@ weight: 300
 ---
 
 # Operators
+River uses a set of operators that most should be familiar with. All operations
+follow the standard [PEMDAS](https://en.wikipedia.org/wiki/Order_of_operations)
+rule for operator precedence.
+
+## Arithmetic operators
+```
++    for addition: 3+5
+-    for subtraction: 10-3
+*    for multiplication: 13*7
+/    for division: 15/3
+%    for remainder: 7%2
+^    for exponentiation: 2^8
+```
+
+The `+` operator can also be used for string concatenation.
+
+## Comparison operators
+```
+==    equal
+!=    not equal
+<     less
+<=    less or equal
+>     greater
+>=    greater or equal
+```
+
+The equality operators `==` and `!=` can be applied to any operands.
+
+On the other hand, for the ordering operators the two operands must both be
+_orderable_ and of the same type. These terms and the result of the comparisons
+are defined as follows:
+
+* Boolean values are comparable. Two boolean values are equal if they are
+  either both true or both false.
+* Numerical (integer and floating-point) values are comparable and ordered, in
+  the usual way.
+* String values are comparable and ordered, lexically byte-wise.
+* Objects are comparable. They are are equal if all their fields are equal.
+* Array values are comparable. They are equal if their corresponding elements
+  are equal.
+
+## Logical operators
+```
+&&    conditional AND
+||    conditional OR 
+!     NOT            
+```
+
+Logical operators apply to boolean values and yield a boolean result.
+
+## Assignment operator
+River uses `=` as its assignment operator.
+
+An assignment statement may only assign a single value to a single attribute.
+In assignments, each value must be _assignable_ to the attribute to which it is
+assigned.
```

Comment:
nit: I'm not really sure if this is an "expression / operator" thing, it's more of a block thing. 

(also `=` is used in object literals to assign a value to a key, it's not just for attributes)

Reply:
The original intent of the first sentence here was to say that you cannot have multiple assignments a la `foo, bar = 1, 2`. 

Sorry, I'm not sure what you meant by 'it's more of a block thing' here, could you expand?

Reply:
Ah, sorry, I meant it might be more appropriate to explain the rules of `=` when we talk about blocks and attributes.

Reply:
Do you recommend dropping the whole 'Assignment operator' section and move it to `syntax.md`? I feel that it's somewhat well tied here as we're discussing the different types under the expressions directory.

Reply:
I don't feel too strongly about it, it's your call. I can see good arguments for both sides here